### DISCUSSION TITLE
How to override only some of the colors in the default themes?

### DISCUSSION THREAD

**Discussion by MetRonnie** [vuetifyjs/vuetify#16801] [Q&A] [answered]
The docs talk about adding a custom theme where you specify all of the colours, but what about just using the default light/dark themes and only customising one or a few of the colours?

Vuetify 3

**Comment by KaelWD** [comment, marked as answer]
Just set the colours you want to change, anything missing will be taken from the default config.

**Reply by MetRonnie** [reply]
Good to know thanks. I've noticed also that if you're overriding `primary` you probably want to override `primary-darken-1` too

**Reply by KaelWD** [reply]
Yeah good point. I wonder if we should omit that and just have it autogenerate instead.
